spec/models/user_spec.rb

```diff
@@ -9,13 +9,33 @@
   end
   subject { build_stubbed :user }
 
-  it { should have_one :subscription }
+  it "includes Filterable module" do
+    expect(User.ancestors).to include(Filterable)
+  end
+
+  context "associations" do
+    it { should have_one(:subscription).autosave(true) }
+
+    it { should have_one :karma }
```

Comment:
@diraulo @tansaku thanks for the feedbacks, will be updating it and will let you know.